src/main/java/me/zeroeightsix/kami/module/modules/render/StorageESP.kt

```diff
@@ -50,6 +50,7 @@ object StorageESP : Module() {
     private val filled = register(Settings.booleanBuilder("Filled").withValue(true).withVisibility { page.value == Page.RENDER }.build())
     private val outline = register(Settings.booleanBuilder("Outline").withValue(true).withVisibility { page.value == Page.RENDER }.build())
     private val tracer = register(Settings.booleanBuilder("Tracer").withValue(false).withVisibility { page.value == Page.RENDER }.build())
+    val cull = register(Settings.booleanBuilder("OnlyVisible").withValue(false).withVisibility { page.value == Page.RENDER }.build())
```

Comment:
To be honest it would make more sense to call this just "Culling", as setting descriptions are planned